lib/cl/api.jl

```diff
@@ -66,7 +66,7 @@ function retry_reclaim(f, isfailed)
         end
     end
 
-    ret
+    return ret
```

Comment:
Unrelated formatting changes like that make it hard to review your PR. The formatter should only have run on changed code; did you format the entire codebase?

Reply:
I tried running Runic on the files it complained.